host-configs/llnl/toss_4_x86_64_ib_cray/cce@12.0.3_clang_hip.cmake

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for the rzwhamo cluster at LLNL
+#------------------------------------------------------------------------------
+# This file provides CMake with paths / details for:
+#  C,C++, & Fortran compilers + MPI & HIP
+# using tce wrappers, rather than HPE Cray PE compiler drivers
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# HPE Cray cce@12.0.3 compilers
+#------------------------------------------------------------------------------
+# _blt_tutorial_compiler_config_start
+set(CCE_HOME "/usr/tce/packages/cce-tce/cce-12.0.3")
+set(CMAKE_C_COMPILER   "${CCE_HOME}/bin/craycc" CACHE PATH "")
+set(CMAKE_CXX_COMPILER "${CCE_HOME}/bin/crayCC" CACHE PATH "")
+
+# Fortran support
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+set(CMAKE_Fortran_COMPILER "${CCE_HOME}/bin/crayftn" CACHE PATH "")
+# _blt_tutorial_compiler_config_end
+
+#------------------------------------------------------------------------------
+# MPI Support
+#------------------------------------------------------------------------------
+# _blt_tutorial_mpi_config_start
```

Comment:
>  _blt_tutorial_mpi_config_start

I'd suggest removing these start/end comments from this host-config.
They are related to our sphinx docs in other host-configs.